### PR TITLE
update: Community Services page and Pricing page update

### PR DESCRIPTION
**Pull request by isTravis**
This PR
- creates a static Membership page at /membership
- create an API endpoint to email ourselves the results of the form on the Membership page

It's been a while since I've made new pages or API routes, so I primarily want to make sure I'm not re-introducing any deprecated bad habits.

### CODE REVIEW

File: server/routes/index.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ require('./dashboardCollectionLayout');
 
 /* Routes for PubPub */
 require('./communityCreate'); // Route: '/community/create'
+require('./communityServices'); // Route: '/community/create'
```
Comment (gabestein) [inline server/routes/index.ts:26]: ultra-super-nit: fix the route comment

**Review by gabestein — APPROVED** [review]
I'm not sure who the last person to create a new static route was, but this looks good to me.

File: server/routes/index.ts (hunk 1 above)
Comment (isTravis) [reply server/routes/index.ts:26]: 💯 I'm all for the ultra-super-nits. Nice catch!

**Review by 3mcd — APPROVED** [review]
LGTM!

**Pull request merged**